example/lib/src/file_picker_demo.dart

```diff
@@ -33,11 +55,32 @@ class _FilePickerDemoState extends State<FilePickerDemo> {
         .addListener(() => _extension = _fileExtensionController.text);
   }
 
+  Widget _buildFilePickerResultsWidget(
+      {required int itemCount,
+      required Widget Function(BuildContext, int) itemBuilder}) {
+    final ScrollController scrollController = ScrollController();
+    return Container(
+      height: MediaQuery.of(context).size.height * 0.50,
```

Comment:
Use the inherited aspect of MediaQuery

```suggestion
      height: MediaQuery.sizeOf(context).height * 0.50,
```